app/views/school/welcome_wizard/allocation.html.erb

```diff
@@ -18,17 +17,6 @@
           <li>students in 16 to 19 further education</li>
           <li>students who get free meals in further education</li>
         </ul>
-      <% else %>
-        <p class="govuk-body">Your allocation of <%= @allocation %> is based on:</p>
-
-        <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
-          <li>the number of children in years 3 to 11</li>
```

Comment:
are we sure we want to delete this? it will now always say `students in 16 to 19 further education` what about if a primary school logs in?

Reply:
it still has:
- pupils in years 3 to 11
- pupils who get free school meals

Reply:
oh sorry the diff cut the first 2 items in the list